gopdf.go

```diff
@@ -1151,4 +1152,84 @@ func infodate(t time.Time) string {
 	return ft
 }
 
+// alpha: 		value from 0 (transparent) to 1 (opaque)
+// blendMode:   blend mode, one of the following:
+//          		Normal, Multiply, Screen, Overlay, Darken, Lighten, ColorDodge, ColorBurn,
+//          		HardLight, SoftLight, Difference, Exclusion, Hue, Saturation, Color, Luminosity
+func (gp *GoPdf) SetAlpha(alpha float64, blendModeStr string) error {
+	if alpha < 0.0 || alpha > 1.0 {
+		return errors.Unwrap(fmt.Errorf("alpha value (0.0 - 1.0) is out of range: %.3f", alpha))
+	}
+
+	blendMode, err := getBlendMode(blendModeStr)
+	if err != nil {
+		return err
+	}
+
+	alphaStr := fmt.Sprintf("%.3f", alpha)
+	keyStr := fmt.Sprintf("%s %s", alphaStr, blendMode)
+
+	transparency, ok := gp.curr.transparencyMap[keyStr]
+	if !ok {
+		index := gp.addExtGStateObj(&ExtGStateObj{
+			ca: alpha,
+			CA: alpha,
+			BM: blendMode,
+		})
+
+		transparency = Transparency{
```

Comment:
unknown field 'alpha' and 'blendMode'  in struct `Transparency`
-----
```GO
//current.go line 92
type Transparency struct {
	IndexOfExtGState int
} 
```

Reply:
fixed
please review again